package.json

```diff
@@ -8,7 +8,7 @@
     "build:default": "npm run style:compile && npm run style:theme && npm run style:postCSS && npm run style:assets",
     "build:prod": "cross-env NODE_ENV=production npm run build",
     "deploy": "gh-pages -d storybook-static/",
-    "lint:style": "stylelint '{src,.storybook}/**/*.(sa|sc|c)ss'",
+    "lint:style": "stylelint \"{src,.storybook}/**/*.(sa|sc|c)ss\"",
```

Comment:
is this part of the issue or it happened with autoformatting?

Reply:
@droshev this isn't part of issue, this is windows specific fix, I ran into the problem when I was running `npm run lint:style`

Reply:
@aberikashvili then can you open this changes in a separate PR?